README.rst

```diff
@@ -129,8 +129,8 @@ page or ask us at our active Gitter channel, where we have maintainers from
 all over the world. We appreciate any help!
 
 We also have a
-`newcomer guide <http://coala.io/newcomer>`__
-to help you get started by fixing an issue yourself! If you get stuck anywhere
+`Guide for Newcomers <http://coala.io/newcomer>`__ page
+that will help you in getting started on contributing to Coala! If you get stuck anywhere
```

Comment:
Replace `Coala` with `coala`(remember it's a small `c` *always*)

Moreover this sentence doesn't sound right to me.

Maybe change this to, `... to help you get onboard contributing to coala ...`, something on the similar lines.